src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs

```diff
@@ -270,6 +270,7 @@ internal void UpdateRunStatusMsgBasedOnStates()
             if(IsHomeSpace && !IsCurrentSpace)
             {
                 SetCurrentWarning(NotificationLevel.Mild, string.Empty);
+                DynamoViewModel.FileTrustViewModel.ShowWarningPopup = false;
```

Comment:
From the gif, there will be a flash of trust warning displayed then disappear.. Less desirable, can we hide it even earlier?

Reply:
I dont think we still need this fix here in addition to other changes?